tools/ci_build/github/azure-pipelines/templates/linux-gpu-tensorrt-packaging-pipeline.yml

```diff
@@ -42,9 +42,9 @@ stages:
   variables:
     - name: linux_trt_version
       ${{ if eq(parameters.CudaVersion, '11.8') }}:
-        value: 8.6.1.6-1.cuda11.8
+        value: 10.0.1.6-1.cuda11.8
       ${{ if eq(parameters.CudaVersion, '12.2') }}:
-        value: 8.6.1.6-1.cuda12.0
+        value: 10.0.1.6-1.cuda12.4
```

Comment:
it looks kind of strange that if cuda version equals 12.2 we use the one with 12.4 
is this the mismatch you were referring to?

Reply:
i guess it's because that's the latest version of CUDA 12 that TensorRT 10 was built against?

Reply:
yeah that's true